frontend/src/views/Site.vue

```diff
@@ -137,15 +201,21 @@ export default class SiteView extends Vue {
   instruments: ReducedMetadataResponse[] | null = null
   instrumentsFromLastDays = 30
   allProducts: Product[] | null = null
+  selectedProductId: string[] = []
   mapKey = 0
   busy = false
   getIconUrl = getProductIcon
   devMode = new DevMode()
   dataStatusParser: DataStatusParser | null = null
-
-
+  singleProductView = false
   payload = {developer: this.devMode.activated}
+
   created() {
+    axios
+      .get(`${this.apiUrl}products/`)
+      .then(({data}) => {
+        this.allProducts = data.filter((product: { level: string }) => product.level != '3')
```

Comment:
Eikö tämä voisi olla:

```suggestion
        this.allProducts = data.filter((product: Product) => product.level != '3')
```

Reply:
Totta!